packages/flutter/test/cupertino/route_test.dart

```diff
@@ -624,4 +638,41 @@ void main() {
     expect(tester.getTopLeft(find.text('1')).dx, moreOrLessEquals(0));
     expect(navigatorKey.currentState.userGestureInProgress, false);
   });
+
+  testWidgets('test edge swipe then drop back at ending point works', (WidgetTester tester) async {
+    await tester.pumpWidget(
```

Comment:
Add comment `/// Regression test for https://github.com/flutter/flutter/issues/29596.`?

Reply:
sg